src/gitops/components/application/graph/ApplicationGraphView.tsx

```
@@ -360,12 +340,12 @@ export const ApplicationGraphView: React.FC<{
   const navigate = useNavigate();
 
   const contextMenuParamsRef = React.useRef<ApplicationContextMenuParams>({
-    application,
-    navigate,
-    launchLabelsModal,
-    launchAnnotationsModal,
-    launchDeleteModal,
-    setGroupNodeStates,
+        application,
```

ESLint

[error] Delete `····`

(prettier/prettier)